### PR TITLE
[NUI] View Guide

### PR DESCRIPTION
**Pull request by MathewP1**
### Change Description ###

-Updated NUI View guide to be more readable and to better explain basic concepts (especially relationship between ParentOrigin and PivotPoint)

### CODE REVIEW

File: docs/application/dotnet/guides/nui/view.md [hunk 1]
```diff
@@ -24,7 +26,7 @@ In addition, if the property `ParentOrigin` is changed to **BottomRight**, then
 </tr>
 </table>
 
-The following figure shows three cases of parent `View` and its child `View`. Parent `View` is blue and child `View` is red. In the following examples, all of the red `View` is identically located at (3, 5) of its parent coordinate system that is based on `ParentOrigin` property of the child `View`.
+The following figure shows three cases of parent `View` and its child `View`. Parent `View` is blue and child `View` is red. In the following examples, all of the red `View` is identically located at (3, 5) of its parent coordinate system that is based on `ParentOrigin` property of the child `View`. `PivotPoint` is set to default **TopLeft**.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:29]:
```suggestion 
The following figure shows three cases of parent `View` and its child `View`. The parent `View` is represented by blue color and child `View` is represented by red color. 
In the following examples, all of the red-colored `View` coordinates are identically located at (3, 5) of its parent coordinate system, which is based on `ParentOrigin` property of the child `View`. `PivotPoint` is set to default **TopLeft**:

File: docs/application/dotnet/guides/nui/view.md [hunk 2]
```diff
@@ -10,11 +10,13 @@
...
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View`'s **TopLeft** corner will be located (5, 3) from **BottomRight**. 
 
-**Figure: The position of predefined ParentOrigin**
+NUI provides predefined `ParentOrigin` as well as `PivotPoint` as shown in the following figure. Alternatively, you can use custom values.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:17]:
```suggestion
NUI provides predefined `ParentOrigin` as well as `PivotPoint` as shown in the following figure. You can also use custom values:

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Request you to kindly incorporate the comments, and also, provide your opinion to some comments inline.

File: docs/application/dotnet/guides/nui/view.md (hunk 2 above)
Comment (MathewP1) [reply docs/application/dotnet/guides/nui/view.md]: I've made changes similar to those you proposed. Please check if that's fine.

File: docs/application/dotnet/guides/nui/view.md [hunk 3]
```diff
@@ -10,11 +10,13 @@
...
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a propetry that defines the anchor point of the child, the default value is **TopLeft**. For example, if a `View`'s property `Position` is set to (5, 3), `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent `View` object.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:13]:
```suggestion 
`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child, the default value is **TopLeft**. For example, if `Position` property of `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent `View` object.

File: docs/application/dotnet/guides/nui/view.md [hunk 4]
```diff
@@ -10,11 +10,13 @@
...
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if the property `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:15]:
```suggestion 
In addition, if `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner.

File: docs/application/dotnet/guides/nui/view.md [hunk 5]
```diff
@@ -10,11 +10,13 @@
...
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if the property `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner. 
 
-**Figure: The position of predefined ParentOrigin**
+NUI provides predefined `ParentOrigin` as well as `PivotPoint` as shown in the following figure. You can also use custom values.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:17]:
```suggestion 
```suggestion
NUI provides predefined `ParentOrigin` as well as `PivotPoint` positions as shown in the following figure, and you can also use custom values for setting the position:
```

I have done a slight modification, request you to kindly check for technical accuracy.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Request you to incorporate the comments. 
Thanks.

File: docs/application/dotnet/guides/nui/view.md (hunk 5 above)
Comment (MathewP1) [reply docs/application/dotnet/guides/nui/view.md]: I have incorporated your suggestions and fixed mistakes.

File: docs/application/dotnet/guides/nui/view.md [hunk 6]
```diff
@@ -10,11 +10,13 @@
...
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a propetry that defines the anchor point of the child, the default value is **TopLeft**. For example, if `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent `View` object.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:13]: @MathewP1 sorry but you seem to have missed correcting spell. 

```suggestion 
`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child, the default value is **TopLeft**. For example, if `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center**, and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent `View` object.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Slight misses, kindly check the spelling of **property**.

**Comment by nawab-samsung** [comment]
@MathewP1 request you to kindly incorporate the comments.

File: docs/application/dotnet/guides/nui/view.md (hunk 6 above)
Comment (MathewP1) [reply docs/application/dotnet/guides/nui/view.md]: I fixed "property" spelling, my bad.

**Review by nawab-samsung — APPROVED** [review]
@MathewP1 thanks for incorporating the comments.

File: docs/application/dotnet/guides/nui/imageview.md [hunk 7]
```diff
@@ -1,53 +1,117 @@
...
+    } 
+    ```
+
+2. Add handler to `ImageView` by using the `ResourceReady` property:
+
+    ```csharp
+    imageView.ResourceReady += onResourceReady;
+    ```
+
+## ImageView Properties
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:95]: ## ImageView properties

File: docs/application/dotnet/guides/nui/imageview.md [hunk 8]
```diff
@@ -1,53 +1,117 @@
...
-<a name="properties"></a>
-## Image View Properties
+3.  To rotate the image by 45 degrees, set the `Orientation` property as follows:
+
+    ```csharp
+    imageView.Orientation = new Rotation(new Radian((float)System.Math.PI / 4), 
+                                         new Vector3(0.0f, 0.0f, 1.0f));
+    ```
+
+## ImageView Event
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:76]: ## ImageView event

File: docs/application/dotnet/guides/nui/imageview.md [hunk 9]
```diff
@@ -1,53 +1,117 @@
...
 
-The following table defines the [Tizen.NUI.BaseComponents.ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class control properties.
+The following table defines the `ImageView` class control properties.
 
-**Table: Image view control properties**
+**Table: ImageView control properties**
 
 | Property             | Type        | Description                              |
 |--------------------|-----------|----------------------------------------|
 | `ResourceUrl`        | `string`    | The file path of the Image.        |
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/imageview.md:104]: The file path of the image.

Any reason why I in Image is mentioned in caps?

**Comment by annie-samsung** [comment]
@MathewP1, the commits need to be latest files, could you please check on that?

**Comment by nawab-samsung** [comment]
@annie-samsung thanks, will check the file for the comments.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Request you to incorporate the comments. 
Global: Please align the images properly w.r.t the text.  
Thanks,

File: docs/application/dotnet/guides/nui/imageview.md (hunk 7 above)
Comment (MathewP1) [reply docs/application/dotnet/guides/nui/imageview.md:95]: Hi, I don't know what's up but there should be only one commit in this PR with changes to view.md, I didn't make any changes to other files, I'm confused what has happened.

File: docs/application/dotnet/guides/nui/view.md [hunk 10]
```diff
@@ -10,11 +10,13 @@
...
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child, the default value is **TopLeft**. For example, if `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent `View` object.
```
[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/view.md:13]: @MathewP1 

is there any reson, ypu didnt incorporte this?

File: docs/application/dotnet/guides/nui/view.md [hunk 11]
```diff
@@ -10,11 +10,13 @@
...
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner. 
 
-**Figure: The position of predefined ParentOrigin**
+NUI provides predefined `ParentOrigin` as well as `PivotPoint` positions as shown in the following figure and you can also use custom values for setting the position:
```
[the remark this answers is not in the document]
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/view.md:17]: @MathewP1

is there any reson, ypu didnt incorporte this?

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
Please incorporate the missed comments.

File: docs/application/dotnet/guides/nui/view.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (MathewP1) [reply docs/application/dotnet/guides/nui/view.md]: I just did that

**Review by nawab-samsung — APPROVED** [review]
@MathewP1 thanks or incorporating the comments.

File: docs/application/dotnet/guides/nui/view.md [hunk 12]
```diff
@@ -10,11 +10,13 @@
...
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child, the default value is **TopLeft**, for instance, if the `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/view.md:13]: @nawab-samsung, please ensure you break such sentences for readability:

The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child.  The default value is **TopLeft**.  
For example, if the `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.

OR

The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child.  The default value is **TopLeft**.  

For example, if the :
- `Position` property of the `View` is set to (5, 3) coordinates
- `ParentOrigin` property is set to **Center** 
- `PivotPoint` property is set to **TopLeft**, 
 
Then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.

Question: 
 The default value is **TopLeft**.  Default value of what? ParentOrigin or PivotPoint?

Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/view.md:13]: @MathewP1

Request you to kindly answer  the question.

Comment (MathewP1) [reply docs/application/dotnet/guides/nui/view.md]: Default is **TopLeft** for both. I've added this to guide.

File: docs/application/dotnet/guides/nui/view.md [hunk 13]
```diff
@@ -10,11 +10,13 @@
...
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child. The default value for both `PivotPoint` and `ParentOrigin` is **TopLeft**. For instance, if the `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:13]: Added Line spaces to add readability
```suggestion
Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. 

The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child. The default value for both `PivotPoint` and `ParentOrigin` is **TopLeft**. For instance, if the `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Please incorporate the minor comment.

File: docs/application/dotnet/guides/nui/view.md [hunk 14]
```diff
@@ -133,4 +135,3 @@ private bool ViewTouchEventCallBack(object sender, View.TouchEventArgs e)
 ## Related Information
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:135]: @MathewP1 Request you to change the headings style to sentence case? 
Please refer to: 
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization

Is it fine for you? 
So, 

 ## Related Information 

becomes 

 ## Related information

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Request you to please change the heading styles.

File: docs/application/dotnet/guides/nui/view.md (hunk 14 above)
Comment (MathewP1) [reply docs/application/dotnet/guides/nui/view.md]: I've changed heading style according to your suggestion

File: docs/application/dotnet/guides/nui/view.md (hunk 13 above)
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:13]:
```suggestion
The `Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child. The default value for both `PivotPoint` and `ParentOrigin` is **TopLeft**. For instance, if the `Position` property of the `View` is set to (5, 3) coordinates, the `ParentOrigin` property is set to **Center**, and the `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.

File: docs/application/dotnet/guides/nui/view.md [hunk 15]
```diff
@@ -10,11 +10,13 @@
...
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:15]:
```suggestion
In addition, if the `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner.

File: docs/application/dotnet/guides/nui/view.md [hunk 16]
```diff
@@ -10,11 +10,13 @@
...
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner. 
 
-**Figure: The position of predefined ParentOrigin**
+NUI provides predefined `ParentOrigin` as well as `PivotPoint` positions as shown in the following figure and you can also use custom values for setting the position:
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:17]:
```suggestion
NUI provides predefined `ParentOrigin` as well as `PivotPoint` positions as shown in the following figure, and you can also use custom values for setting the position:

File: docs/application/dotnet/guides/nui/view.md [hunk 17]
```diff
@@ -24,7 +26,8 @@ In addition, if the property `ParentOrigin` is changed to **BottomRight**, then
 </tr>
 </table>
 
-The following figure shows three cases of parent `View` and its child `View`. Parent `View` is blue and child `View` is red. In the following examples, all of the red `View` is identically located at (3, 5) of its parent coordinate system that is based on `ParentOrigin` property of the child `View`.
+The following figure shows three cases of parent `View` and its child `View`. The parent `View` is represented by blue color and child `View` is represented by red color.
+In the following examples, all of the red-colored `View` coordinates are identically located at (3, 5) of its parent coordinate system, which is based on `ParentOrigin` property of the child `View`. `PivotPoint` is set to default **TopLeft**:
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:30]:
```suggestion
The following figure shows three cases of parent `View` and its child `View`. The parent `View` is represented by blue color and child `View` is represented by red color.
In the following examples, all of the red-colored `View` coordinates are identically located at (3, 5) of its parent coordinate system, which is based on the `ParentOrigin` property of the child `View`. `PivotPoint` is set to default **TopLeft**:

File: docs/application/dotnet/guides/nui/view.md [hunk 18]
```diff
@@ -38,7 +41,7 @@ The following figure shows three cases of parent `View` and its child `View`. Pa
 
 Alternatively, you can use an arbitrary `Position` type value defined in the unit coordinates to set `ParentOrigin` without predefined values.
 
-`Orientation` of `View` is the rotation from its default orientation. `Scale` of `View` is the size ratio between the size to be rendered and the default size. For these two transforms, `View` is rotated and scaled around [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint). NUI also provides predefined presets of `PivotPoint` identical to `ParentOrigin`. Alternatively, you can use custom values.
+`Orientation` of `View` is the rotation from its default orientation. `Scale` of `View` is the size ratio between the size to be rendered and the default size. For these two transforms, `View` is rotated and scaled around `PivotPoint`.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/nui/view.md:44]:
```suggestion
The `Orientation` of `View` is the rotation from its default orientation. `Scale` of `View` is the size ratio between the size to be rendered and the default size. For these two transforms, `View` is rotated and scaled around `PivotPoint`.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@MathewP1 Some minor suggestions that were missed, I am reposting the comments for the same. Rest looks fine.
Thanks,

File: docs/application/dotnet/guides/nui/view.md (hunk 14 above)
Comment (nawab-samsung) [reply docs/application/dotnet/guides/nui/view.md]: @mathewP1 thanks

**Comment by nawab-samsung** [comment]
@MathewP1 Request you to incorporate the comments.

**Comment by MathewP1** [comment]
> @MathewP1 Request you to incorporate the comments.

I have incorporated your comments.

**Review by nawab-samsung — APPROVED** [review]
@MathewP1 thanks for incorporating all the comments.

**Pull request merged**